sites/fast-website/package.json

```diff
@@ -17,9 +17,8 @@
   "devDependencies": {
     "@rollup/plugin-alias": "^3.1.1",
     "clean-webpack-plugin": "^4.0.0",
-    "ejs-loader": "^0.5.0",
-    "html-webpack-plugin": "^5.5.0",
-    "mini-css-extract-plugin": "^2.6.0",
```

Comment:
the modern html-bundler-webpack-plugin replaces functionality of the mini-css-extract-plugin, it is not need more